### PR TITLE
Updates to latest nightly. Fixes #13

### LINKED ISSUE #13: "There is no `AsOsStr` in `std::ffi`" when building
`cargo build` gives `There is no 'AsOsStr' in 'std::ffi'`:

```
cargo build
    Updating registry `https://github.com/rust-lang/crates.io-index`
 Downloading libc v0.1.5
 Downloading docopt v0.6.59
 Downloading regex v0.1.26
 Downloading time v0.1.24
 Downloading inotify v0.1.9
 Downloading rustc-serialize v0.3.12
 Downloading gcc v0.3.4
 Downloading docopt_macros v0.6.59
 Downloading notify v1.2.1
   Compiling inotify v0.1.9
   Compiling rustc-serialize v0.3.12
   Compiling time v0.1.24
error: invalid character `-` in crate name: `build-script-build`
error: invalid character `-` in crate name: `build-script-build`
error: aborting due to 2 previous errors
Build failed, waiting for other jobs to finish...
/home/miguel/.cargo/registry/src/github.com-1ecc6299db9ec823/inotify-0.1.9/src/wrapper.rs:23:5: 23:22 error: unresolved import `std::ffi::AsOsStr`. There is no `AsOsStr` in `std::ffi`
/home/miguel/.cargo/registry/src/github.com-1ecc6299db9ec823/inotify-0.1.9/src/wrapper.rs:23 use std::ffi::AsOsStr;
                                                                                                 ^~~~~~~~~~~~~~~~~
error: aborting due to previous error
error: invalid character `-` in crate name: `rustc-serialize`
error: aborting due to previous error
Could not compile `time`.
```

I am brand new on both Rust and Cargo but [it seems `AsOsStr` has vanished from latest Rust](https://doc.rust-lang.org/std/ffi/)

cargo 0.0.1-pre-nightly (66849de 2015-03-10) (built 2015-03-11)
rustc 1.0.0-dev (built 2015-04-27)

### PR DESCRIPTION
**Pull request by Detegr**

### CODE REVIEW

**Comment by Detegr** [comment]
Changes to `Cargo.lock` make a lot of noise to this PR. If I'm not mistaken it could be removed completely from the repository.

File: Cargo.lock [hunk 1]
```diff
@@ -2,71 +2,94 @@
...
 dependencies = [
- "docopt 0.6.59 (registry+https://github.com/rust-lang/crates.io-index)",
- "docopt_macros 0.6.59 (registry+https://github.com/rust-lang/crates.io-index)",
- "notify 1.2.1 (registry+https://github.com/rust-lang/crates.io-index)",
- "regex 0.1.26 (registry+https://github.com/rust-lang/crates.io-index)",
- "rustc-serialize 0.3.12 (registry+https://github.com/rust-lang/crates.io-index)",
- "time 0.1.24 (registry+https://github.com/rust-lang/crates.io-index)",
+ "docopt 0.6.66 (registry+https://github.com/rust-lang/crates.io-index)",
+ "docopt_macros 0.6.66 (registry+https://github.com/rust-lang/crates.io-index)",
+ "notify 1.2.2 (registry+https://github.com/rust-lang/crates.io-index)",
```
Comment (passcod) [inline Cargo.lock]: Notify is now v2.0.0, probably want to change that in the Cargo.toml. API hasn't changed (the breaking change is not API-related) so should compile fine.

**Comment by passcod** [comment]
Is this still relevant? With the latest release, cargo watch compiles under Rust stable.

**Comment by Detegr** [comment]
It seems these have been fixed along with other changes already. Closing this PR.

**Pull request closed without merging**